15_3_Testing_with_Regtest.md

```diff
@@ -0,0 +1,142 @@
+# 15.3: Testing with Regtest
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+This document explains how to test transactions and situations using regtest (regression test).
+
+
+## Verifying balance
+
+After [mining blocks](15_2_Mining_with_Regtest.md) and getting the rewards, you can verify the balance on your wallet:
+```
+$ bitcoin-cli -regtest getbalance
+50.00000000
+```
+This will print the balance in your wallet.
+
+## Validating the Regtest
+Now you should be able to use this balance for any type of interaction with the private Blockchain, such as sending Bitcoin transactions according to [Chapter 4]((04_0_Sending_Bitcoin_Transactions.md)) in this guide. The only difference is that you need to use the flag `-regtest` when running the `bitcoin-cli` in order for the request to be sent to the Regtest Bitcoin daemon.
+
+It is important to note that for your transactions to complete, you will have to generate/mine new blocks so that the transactions can be included into them.
```

Comment:
Maybe use `for your transactions to confirm` here -- I think the user should know what that means at this point (Chapter 15).